lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -2017,7 +2017,7 @@ public function isIdentifierUuid()
      *
      * @param string $fieldName
      *
-     * @return \Doctrine\DBAL\Types\Type|string|null
```

Comment:
Is this change necessary/correct? Doesn't it affect `ClassMetadataInfo#getTypeOfColumn()` too?

Reply:
It is correct though, since this endpoint always returns a `string`, and the code below does indeed rely on a `string` being retrieved.

If a DBAL type was returned (which is never the case), a string cast would not be acceptable.